web/lua/opmserver/paginator.lua

```diff
@@ -0,0 +1,177 @@
+
+local _M = {}
+local mt = { __index = _M }
+
+local str_find = string.find
+local str_gsub = string.gsub
+local tab_concat = table.concat
+local abs = math.abs
+local modf = math.modf
+local ceil = math.ceil
+
+local function tab_append(t, v, idx)
```

Comment:
```
local function tab_append(t, idx, ...)
    local n = select("#", ...)
    for i = 1, n do
        t[idx + i] = select(i, ...)
    end
    return idx + n
end
```
we can support multiple arguments here, so that we can avoid the string concat thing like:
https://github.com/openresty/opm/pull/65/files#diff-afd0d389fcfb389e98347258a713142cR111